main.go

```diff
@@ -28,6 +28,7 @@ var regArr []*regexp.Regexp
 var nInvertParameter = flag.Bool("v", false, "use PATTERN as non-matching lines.")
 var quietParameter = flag.Bool("q", false, "suppress all normal output")
 var maxCountParameter = flag.Int("m", -1, "stop after NUM selected lines.")
+var lineNumberParameter = flag.Bool("n", false, "print line number before file name and matches.")
```

Comment:
"before file name"?

Reply:
i noticed it later but forgot to change it. Would "print line number with output" be enough?

Reply:
"print" implies "output", so that's even too long.

Reply:
so just "print line number"?

Reply:
Yes.